### DISCUSSION TITLE
Mogrify PDF to TIFF Compression Help

### DISCUSSION THREAD

**Discussion by yngfchn** [ImageMagick/ImageMagick#2304] [Help]
I am using ImageMagick and Ghostscript on Command Prompt.

I am working on converting weekly imported email PDF attachments (1000+) and I need to convert them into TIFF. Another goal is to save space on our servers. However, the results yield TIFF file sizes 10x larger than PDF.

In CMD, I wrote `mogrify -format tiff *.pdf`

How do I compress my TIFF using ImageMagick through CMD? Any alternatives?

**Comment by fmw42** [comment]
See -compress at https://imagemagick.org/script/command-line-options.php#compress. Also see https://imagemagick.org/Usage/formats/#tiff.

**Comment by snibgo** [comment]
PDF files often contain simple text. For example, the string "hello world" would need 11 bytes in the file. ImageMagick is a raster processor, so it will rasterize the PDF file, converting it to an image of pixels, which might need millions of bytes.

Hence, "Another goal is to save space on our servers" may not succeed.
